src/backends/backend/triton_model.cc

```diff
@@ -527,6 +527,33 @@ TRITONBACKEND_RequestCorrelationId(TRITONBACKEND_Request* request, uint64_t* id)
   return nullptr;  // success
 }
 
+TRITONSERVER_Error*
+TRITONBACKEND_RequestStartsSequence(
```

Comment:
The way backends are supposed to know sequence start and end is to use the control inputs. How are they determining this in their current code? We could expose the request flags as a single TRITONBACKEND_RequestFlags function.

Reply:
I see.. I will revert back to the control inputs...